drivers/pinctrl/Kconfig.hpmicro

```diff
@@ -0,0 +1,12 @@
+# Copyright 2022 hpmicro
+# SPDX-License-Identifier: Apache-2.0
+
+DT_COMPAT_HPM_PINCTRL := hpmicro,hpm-pinctrl
+
+config PINCTRL_HPMICRO
+	bool "Pin controller driver for HPMicro MCUs"
+	depends on (SOC_SERIES_HPMICRO)
+	depends on (HAS_HPMSDK_PINCTRL)
```

Comment:
parentheses are redundant here. note that the HAL options are tipically named "USE_" as mentioned earlier.